azure-client-runtime/src/main/java/com/microsoft/azure/v2/CompletedPollStrategy.java

```diff
@@ -20,30 +21,29 @@
  * further polling.
  */
 public class CompletedPollStrategy extends PollStrategy {
-    private final HttpResponse bufferedOriginalHttpResponse;
+    private final BufferedHttpResponse firstHttpResponse;
 
     /**
      * Create a new CompletedPollStrategy.
      * @param restProxy The RestProxy that created this PollStrategy.
      * @param methodParser The method parser that describes the service interface method that
      *                     initiated the long running operation.
-     * @param originalHttpResponse The HTTP response to the original HTTP request.
+     * @param firstHttpResponse The HTTP response to the original HTTP request.
      */
-    public CompletedPollStrategy(RestProxy restProxy, SwaggerMethodParser methodParser, HttpResponse originalHttpResponse) {
+    public CompletedPollStrategy(RestProxy restProxy, SwaggerMethodParser methodParser, HttpResponse firstHttpResponse) {
         super(restProxy, methodParser, 0);
-
-        this.bufferedOriginalHttpResponse = originalHttpResponse.buffer();
+        this.firstHttpResponse = firstHttpResponse.buffer();
```

Comment:
This isn't actually sound if the httpResponse passed were not already buffered--will probably fix this up when it comes time to revisit LROs.